Sejima/Source/MUOverlay/MUOverlay.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  MUOverlay.swift
+//  MUComponent
+//
+//  Created by Loïc GRIFFIE on 31/10/2018.
+//  Copyright © 2018 Loïc GRIFFIE. All rights reserved.
+//
+
+import UIKit
+
+/// Class that generate overlay with customizable options.
+@IBDesignable
+open class MUOverlay: MUNibView {
+    @IBOutlet private var overlay: UIView!
+
+    /// Describes the Overlay's border width appearance while it shows
+    @IBInspectable open dynamic var borderWidth: CGFloat = 2 {
+        didSet {
+            overlay.layer.borderWidth = borderWidth
+        }
+    }
+
+    /// Describes the Overlay's border color appearance while it shows
+    @IBInspectable open dynamic var borderColor: UIColor = .white {
+        didSet {
+            overlay.layer.borderColor = borderColor.cgColor
+        }
+    }
+
+    /// The overlay's style.
+    open var style: MUCornerStyle = .round {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    /// Optional: The IBInspectable version of the overlay's style.
+    @IBInspectable open dynamic var styleInt: Int = 1 {
+        didSet {
+            switch styleInt {
+            case 0:
+                style = .square
+            case 1:
+                style = .round
+            default:
+                style = .custom(radius)
+            }
+        }
+    }
+
+    /// The overlay's corner radius.
+    @IBInspectable open dynamic var radius: CGFloat = 0.0 {
+        didSet {
+            guard radius > 0 else {
+                return
+            }
+            style = .custom(radius)
+        }
+    }
+
+    // MARK: - Private functions
+
+    private func updateStyle() {
```

Comment:
If this function is duplicated in other views, should be nice to add an extension of UIView + MUCornerStyle.

Reply:
It's not possible to add properties in extension